src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,158 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head = null;
+    private Node<T> tail = null;
+
     @Override
     public void add(T value) {
+        Node<T> node;
+        node = new Node<>(null, value,null);
+        if (head == null) {
+            head = node;
+        } else {
```

Comment:
According to the checklist, when creating a new `Node<T>`, pass all the fields to the `Node` constructor. In this case, you should pass the `tail` as the previous node when creating a new node.